eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/io/DbsnpJsonClusteredVariantsWriter.java

```diff
@@ -51,16 +49,18 @@ public DbsnpJsonClusteredVariantsWriter(MongoTemplate mongoTemplate, ImportCount
     @Override
     public void write(List<? extends DbsnpClusteredVariantEntity> clusteredVariants) {
         try {
-            dbsnpClusteredVariantWriter.write(clusteredVariants);
+            if (!clusteredVariants.isEmpty()) {
```

Comment:
Would it be good to log a warning message to point that there were no variants to write?

Reply:
agree